src/semidiscretization/semidiscretization.jl

```diff
@@ -46,10 +46,20 @@ function integrate(func::Func, u_ode, semi::AbstractSemidiscretization; normaliz
   integrate(func, u, mesh, equations, solver, cache, normalize=normalize)
 end
 
-function integrate(u, semi::AbstractSemidiscretization; normalize=true)
-  integrate(cons2cons, u, semi; normalize=normalize)
+function integrate(u_ode, semi::AbstractSemidiscretization; normalize=true)
+  integrate(cons2cons, u_ode, semi; normalize=normalize)
```

Comment:
Be somewhat more expressive here.

Reply:
That's fine with me. Since I think the other parts may be more controversial, you can consider making this a PR on its own.